Components/Atoms/MetricNeomorph.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, Fragment } from "react";
+import { StyleSheet, Text, View, Button, Dimensions, TouchableWithoutFeedback } from "react-native";
+import { Neomorph } from "react-native-neomorph-shadows";
+import Theme from "./Theme";
+
+const MetricNeomorph = ({ text, number }) => {
+  return (
+    <Neomorph darkShadowColor="#D1CDC7" lightShadowColor="#FFF" style={styles.neomorph}>
```

Comment:
I like these metrics, but the neomorph button style made me think these were meant to be clickable

Reply:
good call - yea, the dashboard will need a face lift after brittney's design. i focused on underlying functions rather than look because i know we are going to have to revamp anyway after getting brittney's design